code/datums/traits/neutral.dm

```diff
@@ -103,14 +103,13 @@
 	lose_text = "<span class='notice'>You don't feel that burning sensation anymore.</span>"
 
 /datum/quirk/libido/add()
-	var/mob/living/M = quirk_holder
-	M.min_arousal = 16
-	M.arousal_rate = 3
+	quirk_holder.min_arousal += 16
+	quirk_holder.arousal_rate *= 3
```

Comment:
why change this to += and *= instead of just having it be the previous direct assignment? the *= makes sense if e.g. arousal rate is changed in the future but the += seems weirder than e.g. going with floor((max_arousal-min_arousal)/6)

mostly questioning since the PR says it's just fixing runtimes, and i'm aware this changes absolutely nothing about the code's functionality

Reply:
You are right, this is pretty out of scope and just a grudge agains the = operator.